app/main.py

```diff
@@ -2,0 +1,48 @@
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.append(skill)
+
+
+class FrontendDeveloper(SoftwareEngineer):
+    def __init__(self, name: str) -> None:
+        super().__init__(name)
+        self.skills = ["JavaScript", "HTML", "CSS"]
```

Comment:
The `skills` attribute is re-initialized in each subclass. Consider using the `learn_skill` method to add skills specific to each subclass to avoid redundancy and maintain consistency.